packages/react-instantsearch/examples/next-app/pages/index.js

```diff
@@ -28,9 +28,9 @@ export default class extends React.Component {
      to initialize the searchState. 
   */
   static async getInitialProps(params) {
-    const searchState = qs.parse(
-      params.asPath.substring(params.asPath.indexOf('?') + 1)
-    );
+    const searchState = params.asPath.includes('?')
```

Comment:
.includes is not supported everywhere we support. I think you should do `indexOf('?') > -1` 😄